pkg/middlewares/aws_default_api_gateway_account.go

```diff
@@ -0,0 +1,52 @@
+package middlewares
+
+import (
+	"github.com/sirupsen/logrus"
+	"github.com/snyk/driftctl/pkg/resource"
+	"github.com/snyk/driftctl/pkg/resource/aws"
+)
+
+// AwsDefaultApiGatewayAccount is a middleware that ignores default API gateway account resources in the current region
```

Comment:
We can only have 1 api gateway account, can you change to `that ignores the default API Gateway account resource in the current region`